doc/doxygen/Makefile

```diff
@@ -10,13 +10,23 @@ doc: html
 
 # by marking html as phony we force make to re-run Doxygen even if the directory exists.
 .PHONY: html
-html:
+html: src/css/riot.css
 	( cat riot.doxyfile ; echo "GENERATE_HTML = yes" ) | doxygen -
 
 .PHONY: man
 man:
 	( cat riot.doxyfile ; echo "GENERATE_MAN = yes" ) | doxygen -
 
+ifneq (,$(shell which lessc))
```

Comment:
Can we add a warning message when this tool is not available ? Otherwise, one can change the riot.less file without seeing any change.